src/modules/repository/implementation/sequelize/sequelize-repository.js

```diff
@@ -530,35 +485,21 @@ class SequelizeRepository {
         return abilities.map((e) => e.name);
     }
 
-    async insertBlockchainEvents(blockchainEvents) {
-        const inserted = [];
-
-        await Promise.all(
-            blockchainEvents.map(async (event) => {
-                const exists = await this.blockchainEventExists(
-                    event.contract,
-                    event.event,
-                    event.data,
-                    event.block,
-                    event.blockchainId,
-                );
-                if (!exists) {
-                    const insertionResult = await this.models.blockchain_event.create({
-                        contract: event.contract,
-                        event: event.event,
-                        data: event.data,
-                        block: event.block,
-                        blockchain_id: event.blockchainId,
-                        processed: 0,
-                    });
-                    if (insertionResult?.dataValues) {
-                        inserted.push(insertionResult.dataValues);
-                    }
-                }
-            }),
+    async insertBlockchainEvents(events) {
```

Comment:
@djordjekovac This has to be reviewed. From what I understand, there is no way in sequelize to get only the newly inserted rows. 
What we did previously creates a very large amount of requests to the db and should probably be avoided.
We need to ensure at least one of two things here:

1. we make sure events are never handled twice somehow.
2. we ensure that handling the same event twice doesn't cause any issues.